### ISSUE TITLE
legalcode pages of CC BY Canada versions 2.0 and 2.5 not found

### ISSUE THREAD

**Issue by gho87** [creativecommons/creativecommons.org#750]
I went to https://creativecommons.org/licenses/by/2.5/ca/ (CC BY 2.5 Canada) and clicked both English and French licenses:
* https://creativecommons.org/licenses/by/2.5/ca/legalcode.en
* https://creativecommons.org/licenses/by/2.5/ca/legalcode.fr

Both are "Not Found", i.e. probably no longer working(?).

Same for https://creativecommons.org/licenses/by/2.0/ca/ (CC BY 2.0 Canada):
* https://creativecommons.org/licenses/by/2.5/ca/legalcode.en
* https://creativecommons.org/licenses/by/2.5/ca/legalcode.fr

On the other hand, https://creativecommons.org/licenses/by-sa/2.5/ca/ (CC BY-SA 2.5 Canada) is fine as well as its English and French licenses:
* https://creativecommons.org/licenses/by-sa/2.5/ca/legalcode.en
* https://creativecommons.org/licenses/by-sa/2.5/ca/legalcode.fr

In short, I think the issue is the CC BY 2.0 Canada and CC BY 2.5 Canada legalcodes/license pages no longer working.

**Comment by robmyers** [comment]
This has now been fixed. Thank you for reporting it!
